fix some outdated entries in glossary (#10975)
ethereum/ethereum-org-website: src/content/glossary/index.md

### address {#address}

> A string of 64 characters representing an [EOA](#eoa) or [contract](#contract-account) account in Ethereum's state tree. More specifically, it is the rightmost 160 bits of a [Keccak hash](#keccak-256) of an [ECDSA](#ecdsa) [public key](#public-key).

Reviewer: Agree with 40 change here, I think this is related to private key, but address should be 40. 

@konopkja agree that we could have a simpler definition here as well. Going to commit @wackerow changes, but can propose something here after.

> A string of 40 characters representing an [EOA](#eoa) or [contract](#contract-account) account in Ethereum's state tree. More specifically, it is the rightmost 160 bits of a [Keccak hash](#keccak-256) of an [ECDSA](#ecdsa) [public key](#public-key).

Reviewer: ```suggestion
A string of characters starting with `0x` representing an [EOA](#eoa) or [contract](#contract-account) account in Ethereum's state tree. 
```

I think this could be a more simplified version. Let me know what you think @konopkja

Reviewer: ```suggestion
A unique identifier that represents your account. All accounts on Ethereum have a public address that starts with `0x`. This address is sharable and used to receive assets to your account. 
```

@konopkja put together in call

### committee {#committee}

A group of at least 128 [validators](#validator) assigned to validate blocks in each slot. One of the validators in the committee is the aggregator, responsible for aggregating the signatures of all other validators in the committee that agree on an attestation. Committees rotate such that every active validator attests in every [epoch](#epoch). Not to be confused with [sync committee](#sync-committee).

### computational infeasibility {#computational-infeasibility}

A process is computationally infeasible if it would take an impracticably long time (e.g. billions of years) to do it for anyone who might conceivably have an interest in carrying it out.

### consensus {#consensus}

> When a supermajority of nodes on the network all have the same blocks in their local copy of the blockchain. Not to be confused with [consensus rules](#consensus-rules).

Reviewer 2: what is a supermajority? can we explain better?

Reviewer: ```suggestion
When a supermajority (more than 66%) of nodes on the network all have the same blocks in their local copy of the blockchain. Not to be confused with [consensus rules](#consensus-rules).
```

@konopkja adding a change that should address this, let me know what you think

### layer 2 {#layer-2}

> An area of development focused on moving transactions off of Ethereum layer-1 to enable Ethereum to scale. These improvements reduce transaction fees for users.

Reviewer 2: switch layer 1 lingo to mainnet?

Reviewer: ```suggestion
An area of development focused on moving transactions off of Ethereum Mainnet to enable Ethereum to scale. These improvements reduce transaction fees for users.
```

### non-fungible token (NFT) {#nft}

> An instance of one of several contract standards, commonly [ERC-721](developers/docs/standards/tokens/erc-721) or [ERC-1155](developers/docs/standards/tokens/erc-1155). Each token is unique and distinct; they are not interchangeable like ETH and [ERC-20 tokens](#token-standard). NFTs can represent ownership of digital or physical assets.

Reviewer 3: ```suggestion
An instance of one of several contract standards, commonly [ERC-721](/developers/docs/standards/tokens/erc-721/) or [ERC-1155](/developers/docs/standards/tokens/erc-1155/). Each token is unique and distinct; they are not interchangeable like ETH and [ERC-20 tokens](#token-standard). NFTs can represent ownership of digital or physical assets.
```

Reviewer 2: "An instance of one of several contract standards"  is super difficult to comprehend, we need better explanation

Reviewer 2: @corwintines i think the order in your explanation is wrong. It should start with explanation of uniqueness and end with saying there are few versions of these standards.

Reviewer 2: NFTs can represent ownership of digital or physical assets. Each token is unique and distinct; they are not interchangeable like ETH and [ERC-20 tokens](#token-standard). There are two smart contract standards used for NFTs, [ERC-721](/developers/docs/standards/tokens/erc-721/) or [ERC-1155](/developers/docs/standards/tokens/erc-1155/).

> An instance of one of several contract standards, commonly [ERC-721](/developers/docs/standards/tokens/erc-721/) or [ERC-1155](/developers/docs/standards/tokens/erc-1155/). Each token is unique and distinct; they are not interchangeable like ETH and [ERC-20 tokens](#token-standard). NFTs can represent ownership of digital or physical assets.

Reviewer: ```suggestion
A non-fungible token (NFT) is a one-of-a-kind digital asset that represents ownership of a specific item or piece of content. Common contracts for NFT's include [ERC-721](/developers/docs/standards/tokens/erc-721/) or [ERC-1155](/developers/docs/standards/tokens/erc-1155/).
```

@konopkja tried to improve this like you suggested

### reward {#reward}

> An amount of ether [awarded to validators](developers/docs/consensus-mechanisms/pos/rewards-and-penalties) that perform certain functions, including proposing a block or participating in a sync-committee, in each slot.

Reviewer 3: ```suggestion
An amount of ether [awarded to validators](/developers/docs/consensus-mechanisms/pos/rewards-and-penalties/) that perform certain functions, including proposing a block or participating in a sync-committee, in each slot.
```

### Recursive Length Prefix (RLP) {#rlp}

> An [encoding standard](developers/docs/data-structures-and-encoding) designed by the Ethereum developers to encode and serialize objects (data structures) of arbitrary complexity and length.

Reviewer 3: ```suggestion
An [encoding standard](/developers/docs/data-structures-and-encoding/) designed by the Ethereum developers to encode and serialize objects (data structures) of arbitrary complexity and length.
```
